ibis/backends/duckdb/registry.py

```diff
@@ -255,3 +255,29 @@ def _struct_column(t, op):
         ops.MultiQuantile: reduction(sa.func.quantile_cont),
     }
 )
+
+
+_invalid_operations = {
+    # ibis.expr.operations.analytic
+    ops.CumulativeAll,
+    ops.CumulativeAny,
+    ops.CumulativeOp,
+    ops.NTile,
+    # ibis.expr.operations.generic
+    ops.TableArrayView,
+    ops.TypeOf,
+    # ibis.expr.operations.logical
+    ops.ExistsSubquery,
+    ops.NotExistsSubquery,
+    # ibis.expr.operations.reductions
+    ops.CountDistinct,
```

Comment:
I don't think you can remove a bunch of these because duckdb reuses the postgres rules, most of which are valid for duckdb.

Reply:
I run all the backend tests locally and they succeed, so now we should either add tests to cover these operations or delete them because it's dead code. however, I see that the TPC-H job failed, so we should probably add new tests. I'll deal with it tomorrow.

Reply:
<img width="943" alt="Screenshot 2023-01-10 at 21 49 58" src="https://user-images.githubusercontent.com/120908425/211659734-d4ff516c-3959-4354-846e-b2dd87fb0877.png">

Reply:
CI passed.

Reply:
Interesting, yeah then we are lacking coverage! Thanks for finding these, much appreciated.